### ISSUE TITLE
Dropbox support not working

### ISSUE THREAD

**Issue by obbardc** [subdavis/Tusk#152]
## This issue is a 

- [x] bug
- [ ] feature
- [ ] question

### Please describe the current behavior, and explain why it's bad.

I am trying to connect my Dropbox account, upon enabling I get an error:
"Error: Authorization page could not be loaded."

### Anything else?

I am running Debian testing, Chromium version 62.0.3202.89 (Developer Build)

Cheers!

**Comment by subdavis** [comment]
Why are you running chrome 62?

I have no easy way of testing on a build that old, and don't really intend to support it.  

I cannot find any problems on Ubuntu with Chrome 66.0.3359.170

If you can produce on a fresh install of latest chrome, please provide steps to reproduce.

Otherwise, try clearing your cache of everything, reinstalling Tusk, and enabling Dropbox again.  

Thanks.

**Comment by obbardc** [comment]
Hi, I am running Debian testing where Chromium is at version 62. I've just looked and it's horrendously out of date!

Let me try again on version 66 from Debian sid tonight and get back to you :-)

Thanks

**Comment by obbardc** [comment]
Hi @subdavis 

All OK with latest version of chromium in Debian Sid archive! Thanks for your help.

Chris

**Comment by jormercury** [comment]
I have been getting the same error.
